src/CSS.php

```diff
@@ -66,6 +66,29 @@ public function setImportExtensions(array $extensions)
     }
 
     /**
+     * Move any import statements to the top
+     *
+     * @param $content string Nearly finished CSS content
+     *
+     * @return string
+     */
+    protected function moveImportsToTop($content) {
+
+        if (preg_match_all('#@import[^;]+;#', $content, $matches)) {
+
+            // remove from content
+            foreach($matches[0] as $import) {
+                $content = str_replace($import, '', $content);
```

Comment:
You have a point, and moving around with the order of things in CSS is not my favourite kind of thing. However, browsers don't parse @import statements unless they precede all other statements. See: https://developer.mozilla.org/en-US/docs/Web/CSS/@import

In your example the text in the body would still be blue regardless of the second import statement, which simply gets ignored.

The only possible problem I'd see is, if you have a css file that looks like this:

**style.css**
```css
@import url(local.css);
@import url(//example.com/reset.css);
```
**local.css**
```css
body { color: green; }
```
**example.com/reset.css**
```css
body { color: blue; }
h1 { color: teal; }
```

In this case, body text would remain green with my pull request merged. But the alternative is worse; namely that text remains green, and the h1 color remains at the browser default as reset.css never gets read.